locale/action_names.rb

```diff
@@ -0,0 +1,7 @@
+# Autogenerated!
+_("Action with sub plans")
+_("Create architecture")
+_("Delete architecture")
+_("Update architecture")
+_("Import facts")
+_("")
```

Comment:
And it explodes on the given code, so I commented it out to generate this PR, which generated an empty string, I guess.